library/src/main/java/com/pokegoapi/api/map/fort/Pokestop.java

```diff
@@ -261,20 +260,43 @@ public boolean hasLurePokemon() {
 	}
 
 	/**
+	 * Returns whether this pokestop has an active lure when detected on map.
+	 *
+	 * @return lure status
+	 */
+	public boolean hasLure() {
+		try {
+			return hasLure(false);
+		} catch (LoginFailedException e) {
```

Comment:
Mhhhhh does it pass checkstyle then? I can remember that when i catched 2> exceptions in a block the checkstyle says something about old java not supporting it. Maybe i'm wrong